src/core/ColorPicker/ColorPicker.test.tsx

```diff
@@ -730,3 +820,108 @@ it('should not set focus if setFocus is false', () => {
+    </ColorPicker>,
+  );
+
+  expect(container.querySelector(`.iui-color-picker`)).toBeTruthy();
+  expect(container.querySelector(`.iui-color-selection-wrapper`)).toBeTruthy();
+  expect(
+    container.querySelectorAll(`.iui-color-picker-section-label`).length,
+  ).toBe(1);
+  const element = container.querySelectorAll(
+    `.iui-color-picker-section-label`,
+  )[0];
+  expect(element).toBeDefined();
+  expect(element?.textContent).toBe('HSLA');
+  expect(container.querySelector(`.iui-color-field`)).toBeTruthy();
+  expect(container.querySelector(`.iui-hue-slider`)).toBeTruthy();
+  expect(container.querySelector(`.iui-opacity-slider`)).toBeTruthy();
+  expect(container.querySelector(`.iui-color-dot`)).toBeTruthy();
+  expect(container.querySelectorAll('input').length).toBe(4);
+});
+
+it('should handle arrow key navigation on opacity slider dot', () => {
+  const onSelectionChanged = jest.fn();
+  const selectedColor = ColorValue.create({ h: 0, s: 100, l: 50 });
+  const { container } = render(
+    <ColorPicker
+      showAlpha={true}
+      onChangeComplete={onSelectionChanged}
+      selectedColor={selectedColor}
+    >
+      <ColorBuilder />
+    </ColorPicker>,
+  );
+
+  const colorBuilder = container.querySelector(
+    '.iui-color-picker .iui-color-field',
+  ) as HTMLElement;
+  expect(colorBuilder).toHaveStyle('--hue: #ff0000; --selected-color: #ff0000');
+
+  const opacityDot = container.querySelectorAll(
+    '.iui-slider-thumb',
+  )[1] as HTMLElement;
+  expect(opacityDot).toBeTruthy();
+  expect(opacityDot.style.getPropertyValue('left')).toEqual('100%');
+
+  // Go left
+  fireEvent.keyDown(opacityDot, { key: 'ArrowLeft' });
+  expect(onSelectionChanged).toHaveBeenNthCalledWith(
+    1,
+    ColorValue.create({ h: 0, s: 100, l: 50, a: 0.99 }),
+  );
+  expect(opacityDot.style.getPropertyValue('left')).toEqual('99%');
+
+  // Go right
+  fireEvent.keyDown(opacityDot, { key: 'ArrowRight' });
+  expect(onSelectionChanged).toHaveBeenNthCalledWith(
+    2,
+    ColorValue.create({ h: 0, s: 100, l: 50, a: 1 }),
+  );
+  expect(opacityDot.style.getPropertyValue('left')).toEqual('100%');
+});
+
+it('should render color picker when alpha is false', () => {
+  const onSelectionChanged = jest.fn();
+  const selectedColor = ColorValue.create({ h: 0, s: 100, l: 50 });
+  const { container } = render(
+    <ColorPicker
+      showAlpha={false}
+      onChangeComplete={onSelectionChanged}
+      selectedColor={selectedColor}
+    >
+      <ColorBuilder />
+    </ColorPicker>,
+  );
+
+  const colorBuilder = container.querySelector(
+    '.iui-color-picker .iui-color-field',
+  ) as HTMLElement;
+  expect(colorBuilder).toHaveStyle('--hue: #ff0000; --selected-color: #ff0000');
+
+  const hueSlider = container.querySelector('.iui-hue-slider') as HTMLElement;
+  expect(hueSlider).toBeTruthy();
+
+  const opacitySlider = container.querySelector(
+    '.iui-opacity-slider',
+  ) as HTMLElement;
+  expect(opacitySlider).toBeFalsy();
```

Comment:
Most of these don't need to be assigned
```suggestion
  expect(container.querySelector(
    '.iui-color-field',
  )).toHaveStyle('--hue: #ff0000; --selected-color: #ff0000');

  expect(container.querySelector('.iui-hue-slider')).toBeTruthy();

  expect(container.querySelector('.iui-opacity-slider')).toBeFalsy();
```